src/main/java/com/cedarsoftware/util/reflect/filters/field/GroovyFieldFilter.java

```diff
@@ -16,9 +16,4 @@ public class GroovyFieldFilter implements FieldFilter {
     public boolean filter(Field field) {
         return META_CLASS_FIELD_NAME.equals(field.getName()) && META_CLASS_NAME.equals(field.getType().getName());
```

Comment:
return META_CLASS_FIELD_NAME.equals(field.getName()) && META_CLASS_NAME.equals(field.getType().getName());

I am pretty sure this should be an "OR" not a "AND"

Reply:
This has to be an AND because someone could name their field name metaClass (for some reason) and we only want to filter it when its also in one of Groovy's metaclasses.